src/ecosystem.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	apps: [
+		{
+			name: 'mentored-mentoring',
```

Comment:
This for controlling pm2 deployment of this particular services. Good to have for local setup, since it comes with watch mode and all. Can be removed if not necessary. Was quite useful for me for development.